.github/workflows/go.yaml

```diff
@@ -48,10 +48,10 @@ jobs:
           go-version-file: 'go.mod'
           cache: false
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v6
+        uses: golangci/golangci-lint-action@v7
         with:
           only-new-issues: true
-          version: v1.55.2
+          version: v2.0.2
```

Comment:
While here, can you bump the version?

Reply:
Sure!
[upgrade(golangci-lint): v2.0.2 to v2.1.2](https://github.com/actions/actions-runner-controller/pull/4023/commits/2e19e0401597158d119523f0612903eeb903abd1)

Reply:
Sure!
[upgrade(golangci-lint): v2.0.2 to v2.1.2](https://github.com/actions/actions-runner-controller/pull/4023/commits/2e19e0401597158d119523f0612903eeb903abd1)